pkg/install/fixlbprobeconfig.go

```diff
@@ -0,0 +1,76 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-07-01/network"
+
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+)
+
+func (i *Installer) fixLBProbeConfig(ctx context.Context, resourceGroup, lbName string) error {
+	lb, err := i.loadbalancers.Get(ctx, resourceGroup, lbName, "")
+	if err != nil {
+		return err
+	}
+
+	if lb.LoadBalancerPropertiesFormat == nil || lb.LoadBalancerPropertiesFormat.Probes == nil {
+		return nil
+	}
+
+	var changed bool
+
+loop:
```

Comment:
personally really don't like these goto statements

Reply:
me too :/ I still think this is antipattern we should avoid in any code, any language. 
| ... Dijkstra argued that unrestricted GOTO statements should be abolished from higher-level languages because they complicated the task of analyzing and verifying the correctness of programs (particularly those involving loops).

Reply:
That is not a `goto`.  That is a labeled break/continue.